fl4health/clients/mkmmd_clients/ditto_mkmmd_client.py

```diff
@@ -0,0 +1,274 @@
+from logging import ERROR, INFO
+from pathlib import Path
+from typing import Dict, Optional, Sequence, Tuple
+
+import torch
+import torch.nn as nn
+from flwr.common.logger import log
+from flwr.common.typing import Config, Scalar
+
+from fl4health.checkpointing.client_module import CheckpointMode, ClientCheckpointModule
+from fl4health.clients.basic_client import TorchInputType
+from fl4health.clients.ditto_client import DittoClient
+from fl4health.losses.mkmmd_loss import MkMmdLoss
+from fl4health.model_bases.feature_extractor_buffer import FeatureExtractorBuffer
+from fl4health.utils.losses import LossMeterType
+from fl4health.utils.metrics import Metric
+
+
+class DittoMkmmdClient(DittoClient):
```

Comment:
Very minor, the acronym capitalization we used for the loss function was MkMmd (vs. Mkmmd) since its MK-MMD as an acronym and we treated the - as a space? We can do either, I don't really care, but maybe we can be consistent? We can also do the same for MR-MTL as either MrMtl or Mrmtl (I like the first a little better looking at it now)

Reply:
I update these, but MrMtlMkMmd looks so spiky :)

Reply:
Yeah, it doesn't look great 😂. Still, it follows our convention I guess.